xbmc/addons/RepositoryUpdater.cpp

```diff
@@ -43,8 +43,7 @@ namespace ADDON
 
 CRepositoryUpdater::CRepositoryUpdater(CAddonMgr& addonMgr) :
   m_timer(this),
-  m_doneEvent(true),
-  m_addonMgr(addonMgr)
```

Comment:
This variable should be used below instead of `CAddonMgr::GetInstance()` I'm guessing